irohad/network/ordering_service.hpp

```diff
@@ -34,7 +34,8 @@ namespace iroha {
        * Transform model proposal to transport object and send to peers
        * @param proposal - object for propagation
        */
-      virtual void publishProposal(model::Proposal &&proposal) = 0;
+      virtual void publishProposal(
```

Comment:
It does not seem necessary to use concrete transport in interface. PolymorphicWrapper for interface should be fine here.